apps/roam/src/components/LeftSidebarView.tsx

```diff
@@ -574,6 +572,11 @@ const FavouritesPopover = ({ onloadArgs }: { onloadArgs: OnloadArgs }) => {
 const LeftSidebarView = ({ onloadArgs }: { onloadArgs: OnloadArgs }) => {
   const initialConfig = useConfig();
   const [config, setConfig] = useState(initialConfig);
+
+  useEffect(() => {
```

Comment:
Need this to update the state of sections if we add new section or move the sections we want this to render that state